Packs/BmcITSM/Integrations/BmcITSM/BmcITSM_description.md

```diff
@@ -0,0 +1,9 @@
+## Access to BMC Helix ITSM
+- The credentials (username and password parameters) are used to authenticate with the ITSM API for the provided instance URL. 
+
+- The role of the user should have access to service requests, change requests, problem investigation, known error, task and incidents. Keep in mind that in order to delete a ticket the user must have an Admin role. 
+
+## Display ID vs. Request ID
+- **Display ID** (ID number) is a unique identification to reference a specific ticket. This field allows us to understand the ID of the ticket as it displayed in BMC Helix ITSM. 
```

Comment:
- **Display ID** (ID number) is a unique ID that references a specific ticket. This field is comparable to the ticket ID as it displayed in BMC Helix ITSM.